docs_user/modules/proc_migrating-existing-daemons-to-target-nodes.adoc

```diff
@@ -0,0 +1,176 @@
+for item in $(sudo cephadm shell --  ceph orch host ls --format json | jq -r '.[].hostname'); do
+    sudo cephadm shell -- ceph orch host label add  $item monitoring;
+done
+----
+
+. Verify that all the (three) hosts have the monitoring label:
++
+----
+[tripleo-admin@controller-0 ~]$ sudo cephadm shell -- ceph orch host ls
+
+HOST                        ADDR           LABELS
+cephstorage-0.redhat.local  192.168.24.11  osd monitoring
+cephstorage-1.redhat.local  192.168.24.12  osd monitoring
+cephstorage-2.redhat.local  192.168.24.47  osd monitoring
+controller-0.redhat.local   192.168.24.35  _admin mon mgr monitoring
+controller-1.redhat.local   192.168.24.53  mon _admin mgr monitoring
+controller-2.redhat.local   192.168.24.10  mon _admin mgr monitoring
+----
+
+. Remove the labels from the Controller nodes:
++
+----
+$ for i in 0 1 2; do ceph orch host label rm "controller-$i.redhat.local" monitoring; done
+
+Removed label monitoring from host controller-0.redhat.local
+Removed label monitoring from host controller-1.redhat.local
+Removed label monitoring from host controller-2.redhat.local
+----
+
+. Dump the current monitoring stack spec:
++
+----
+function export_spec {
+    local component="$1"
+    local target_dir="$2"
+    sudo cephadm shell -- ceph orch ls --export "$component" > "$target_dir/$component"
+}
+
+SPEC_DIR=${SPEC_DIR:-"$PWD/ceph_specs"}
+for m in grafana prometheus alertmanager; do
+    export_spec "$m" "$SPEC_DIR"
+done
+----
+
+. For each daemon, edit the current spec and replace the placement/hosts section
+with the placement/label section, for example:
++
+[source,yaml]
+----
+service_type: grafana
+service_name: grafana
+placement:
+  label: monitoring
+networks:
+- 172.17.3.0/24
+spec:
+  port: 3100
+----
++
+The same procedure applies to Prometheus and Alertmanager specs.
+
+. Apply the new monitoring spec to relocate the monitoring stack daemons:
++
+----
+SPEC_DIR=${SPEC_DIR:-"$PWD/ceph_specs"}
+function migrate_daemon {
+    local component="$1"
+    local target_dir="$2"
+    sudo cephadm shell -m "$target_dir" -- ceph orch apply -i /mnt/ceph_specs/$component
+}
+for m in grafana prometheus alertmanager; do
+    migrate_daemon  "$m" "$SPEC_DIR"
+done
+----
+
+. Verify that the daemons are deployed on the expected nodes:
++
+----
+[ceph: root@controller-0 /]# ceph orch ps | grep -iE "(prome|alert|grafa)"
+alertmanager.cephstorage-2  cephstorage-2.redhat.local  172.17.3.144:9093,9094
+grafana.cephstorage-0       cephstorage-0.redhat.local  172.17.3.83:3100
+prometheus.cephstorage-1    cephstorage-1.redhat.local  172.17.3.53:9092
+----
++
+[NOTE]
+With the procedure described above we lose High Availability: the monitoring
+stack daemons have no VIP and haproxy anymore; Node exporters are still
+running on all the nodes: instead of using labels we keep the current approach
+as we want to not reduce the monitoring space covered.
+//kgilliga: Same question here: What does "the procedure described above" refer to?
```

Comment:
The "procedure above, from L13 to L99 describes the migration of the monitoring stack, and it's the core part. I think we can change the wording here, or even remove the "With the procedure described above" part. I want to highlight, once again, that people are going to lose something, but if you feel this Note redundant (because I recall a similar one in a different section), we can have just one that capture the same thing.